docs/setup.rst

```diff
@@ -0,0 +1,90 @@
+Setting Up Corobo
+=================
+
+The following documentation illustrates the process to install Corobo and interface it with the gitter API.
+
+Installation
+------------
+
+*  To install corobo, first you need to setup a virtual environment(it is not mandatory but highly recommended). This can be done as follows:
+   ::
+     pip3 install virtualenv
+     virtualenv venv -p python
+     cd venv
+     source bin/activate ---->  For Linux
+     \Scripts\activate   ---->  For Windows
+*  Next, you can clone the corobo package via git and install the requirements(dependent packages).
+   ::
+     git clone https://github.com/coala/corobo.git
+     pip install -r requirements.txt
+
+Configuration of Environment variables(In text mode)
+--------------------------------------------------
+
+*  The following environment variables are required to setup corobo in text mode:
+   ::
+     export BOT_PREFIX="any_name"
+     export BACKEND="Text"
+     export BOT_USERNAME="@any_other_name"
+   (**Note:** The BOT_USERNAME always starts with a @ for a person and # for a room)
```

Comment:
This is incomplete. And what is `BOT_USERNAME`?

Reply:
This was mentioned in the docs. Seems like it is not used now. Will get it removed.
Also, can u pls suggest the points that you mean to get added??